modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-backing-up-pvs-csi_{context}"]
+= Backing up persistent volumes with Container Snapshot Interface snapshots
+
+You back up persistent volumes with Container Snapshot Interface (CSI) snapshots by creating a `VolumeSnapshotClass` CR.
+
+.Prerequisites
+
+* The cloud provider must support CSI snapshots.
+* You must install the OpenShift API for Data Protection (OADP) Operator and the Data Protection Application.
+* You must add the `csi` default plugin and the `EnableCSI` feature flag to the `DataProtectionApplication` manifest:
++
+.Example DataProtectionApplication manifest with CSI
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+...
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - csi
+    featureFlags:
+    - EnableCSI
+----
+
+.Procedure
+
+. Create a `VolumeSnapshotClass` manifest for CSI, as in the following example:
++
```

Comment:
s/CR/manifest ?

Reply:
The CR is created from the manifest. You create a manifest. That creates the CR.